html/excercise_two.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Excersie Two</title>
+  </head>
+  <body>
+    <h2>New Task</h2>
+    <p>Create a new monthly task and keep track of it.</p>
+    <a href="">Back</a>
+    <br>
+    <br>
+    <form>
+      <label>Name: <br></label>
+      <input><br>
```

Comment:
inputs will needs some attributes, such as `type="text"` to let them know the input will have a specific type.
```suggestion
      <input type="text"><br>
```

do this to both of them.